server/controllers/work.js

```diff
@@ -48,6 +78,8 @@ export const work = async(ctx, next) => {
   const imgWidth = '800';
   const imgLink = imageUrlFromMiroId(miroId);
   const encoreLink = sierraId && encoreLinkFromSierraId(sierraId);
+  const canonicalUri = `${ctx.globals.rootDomain}/works/${singleWork.id}`;
```

Comment:
I started removing the rootDomain as it was being duplicated on events etc.
Not sure if it's the same here.